test/unit/services/leafletGeoHelpersSpec.coffee

```diff
@@ -0,0 +1,59 @@
+describe 'leafletGeoJsonHelpers', ->
```

Comment:
specs to show validation of a Point (lat: lng) . Also verify the creation / usage of geojson Point and Array [lng,lat].